devtools/debug-column-mapping.ipynb

```diff
@@ -30,6 +30,16 @@
     "%autoreload 2"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # the specific extractor is imported up above in the import cell.
</pre><p>non-blocking: maybe actually easier to move the import step to this cell. It seems not ideal to have to always remember to change what's getting imported, as you note above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/3681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
yea i think this is a good idea! it's not standard tidy notebook best practices but i think in this case it makes more sense.